modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/RequiredMark.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+const RequiredMark = () => (
+	<>
+		<span className="inline-item-after reference-mark text-warning">
+			<ClayIcon symbol="asterisk" />
+		</span>
+		<span className="hide-accessible sr-only">
+			{Liferay.Language.get('required')}
+		</span>
+	</>
+);
```

Comment:
We have that bug in our formatter so we need to manually add empty lines between sibling elements inside react fragments